### DISCUSSION TITLE
Hold to confirm should not apply to pausing maps

### DISCUSSION THREAD

**Discussion by aljich** [ppy/osu#30305] [Ideas] [answered]
I feel like it's pretty unintuitive to have hold to confirm time for pausing during a map. And yes, you can adjust that timing to be 0, but there are other actions that it makes sense to have a hold to confirm time for, such as quick restarting. Curious as to hear other perspectives about this. It makes sense to me that you shouldn't have to hold a button to pause a map.

**Comment by peppy** [comment, marked as answer]
This has been brought up so many times i'm inclined to change this. We can probably remove the delay from the keyboard pause but leave it on the button.

**Reply by aljich** [reply]
Thanks for the prompt response, glad to hear others are of similar opinion 😄

**Reply by Joehuu** [reply]
Change has been made in https://github.com/ppy/osu/pull/30309.
